src/Components/Login/Footer.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import './Footer.scss';
+
+class Footer extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      footerLogin: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/Jungwoo/footerLogin.json', {
+      method: 'GET',
```

Comment:
```suggestion
    fetch('/data/Jungwoo/footerLogin.json'
```
- local의 public 폴더에 접근할 때는 루트 경로로 시작하면 도메인, 포트 번호 적어주지 않더라도 접근 가능합니다
- get method는 기본값이라서 생략가능해요

Reply:
넵 감사합니다. fetch() 블로그 게시물에 첫번째 부분도 추가해놓겠습니다.